mantidproject/mantid
# Load and display data for PANTHER in diffraction mode

**Description of work.**
This PR makes it possible to load and display data taken with PANTHER using diffraction mode.
Edit : it also makes it possible to run the `SumOverlappingTubes` algorithm on these data.

**To test:**
- Load `ExternalData/Testing/Data/UnitTest/ILL/PANTHER/001036.nxs` on Workbench and check data are correct and are correctly displayed in the instrument view.
- Run `SumOverlappingTubes` on the workspace, with `OutputType` either 2D or 2DTubes, and `ScatteringAngleBinning = 0.508`. Check correctness of the output colorfill.

Fixes #27711


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
